app/email_batching_test.go

```diff
@@ -25,54 +25,36 @@ func TestHandleNewNotifications(t *testing.T) {
 
 	job.handleNewNotifications()
 
-	if len(job.pendingNotifications) != 0 {
-		t.Fatal("shouldn't have added any pending notifications")
-	}
+	require.Equal(t, 0, len(job.pendingNotifications), "shouldn't have added any pending notifications")
 
 	job.Add(&model.User{Id: id1}, &model.Post{UserId: id1, Message: "test"}, &model.Team{Name: "team"})
-	if len(job.pendingNotifications) != 0 {
-		t.Fatal("shouldn't have added any pending notifications")
-	}
+
+	require.Equal(t, 0, len(job.pendingNotifications), "shouldn't have added any pending notifications")
 
 	job.handleNewNotifications()
-	if len(job.pendingNotifications) != 1 {
-		t.Fatal("should have received posts for 1 user")
-	} else if len(job.pendingNotifications[id1]) != 1 {
-		t.Fatal("should have received 1 post for user")
-	}
+	require.NotEqual(t, 1, len(job.pendingNotifications), "should have received posts for 1 user")
```

Comment:
Here should be `Equal`, and in the next line the same

Reply:
Actually, you can use `require.Len` here to be more semantic about what are you testing (This same thing can be applied in other places)